pkg/cmd/add.go

```diff
@@ -2,47 +2,44 @@ package cmd
 
 import (
 	"github.com/MakeNowJust/heredoc"
-	"github.com/spf13/afero"
 	"github.com/spf13/cobra"
-)
 
-var createDirs bool
+	"github.com/evilmartians/lefthook/pkg/lefthook"
+)
 
-var longAddCmd = heredoc.Doc(`
-	This command will try to build the following structure in repository:
-	├───.git
-	│   └───hooks
-	│       └───pre-commit // this executable will be added. Existed file with
-	│                      // same name will be renamed to pre-commit.old
-	(lefthook add this dirs if you run command with -d option)
-	│
-	├───.lefthook          // directory for project level hooks
-	│   └───pre-commit     // directory with hooks executables
-	├───.lefthook-local    // directory for personal hooks add it in .gitignore
-	│   └───pre-commit
-`)
+func NewAddCmd(opts *lefthook.Options) *cobra.Command {
+	args := lefthook.AddArgs{}
 
-func NewAddCmd(rootCmd *cobra.Command) {
-	addCmd := &cobra.Command{
-		Use:   "add",
+	addCmd := cobra.Command{
+		Use:   "add hook-name",
 		Short: "This command add a hook directory to a repository",
-		Long:  longAddCmd,
-		Args:  cobra.MinimumNArgs(1),
-		RunE: func(cmd *cobra.Command, args []string) error {
-			return addExecutor(args, appFs)
+		Long: heredoc.Doc(`
```

Comment:
What about using the `embed` package and put the long description into a text file? https://pkg.go.dev/embed